lib/kino/proxy.ex

```diff
@@ -46,8 +44,28 @@ defmodule Kino.Proxy do
           |> Plug.Conn.send_resp(200, "use /export/data to get extract the report data")
       end)
 
-  Once deployed as an app, the user would be able to export the data
+  Once deployed as an app, the API client would be able to export the data
   by sending a request to `/apps/:slug/proxy/export/data`.
+
+  > #### Authentication {: .warning}
+  >
+  > The paths exposed by `Kino.Proxy` don't use the authentication mechanisms
+  > defined in your Livebook instance.
```

Comment:
@josevalim I started to wonder if we should put them in the `/public` namespace. The main downside is that it differs more from the base app/session path.

Reply:
We can always add more routes in the future. I think we have a good starting set. I can also see it being used for coordinating tasks inside their own infrastructure, but still not exposing it to the world.